Trident.xcodeproj/project.pbxproj

```diff
@@ -158,7 +158,7 @@
 				TargetAttributes = {
 					43CDFA421DCFA16600046EB0 = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = 4P3PNUDM94;
+						DevelopmentTeam = 329FNU8Z9Y;
```

Comment:
I didn't keep this change because it's specific to each person so it will be infinite changes for nothing